### PR TITLE
Hrosenbe/weat 134 simplify host specification

### PR DESCRIPTION
**Pull request by hrosenbe**
This pull request has two changes:
1. Add an appInstanceCluster parameter to specify the kubernetes cluster on which to run an appInstance.  The exisiting parameter appInstanceHost must now refer to a dockerHost.
1. The user can now specify the name of docker hosts in appInstanceHost, webServerHost, etc. that were previously not defined in a dockerHosts parameter.  dockerHosts specified this way use the defaults for the other parameters that apply to dockerHosts: dockerPort, dockerNet, vicHost.

This pull also updates the example config files to reflect this change, and has a minor fix for checking the format of the cpu and mem parameters.

### CODE REVIEW

File: weathervane.config.docker.custom [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionDriver1", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionWeb1", "dockerPort" : 2376, "dockerNet" : "bridge" }, 
+    { "name" : "AuctionBid1", "dockerPort" : 2375, "dockerNet" : "bridge" }, 
+    { "name" : "AuctionBid2", "dockerPort" : 2376, "dockerNet" : "host" }, 
+    { "name" : "AuctionBid3", "dockerPort" : 2376, "dockerNet" : "host" },
```
Comment (bhoflich) [inline weathervane.config.docker.custom]: 1 "bridge" and 2 "host" for AuctionBids?  Is that intended?

File: weathervane.config.docker.micro.oneHost [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "dockerNamespace" : "yourRepository",
+  
+  "numDrivers": 8,
```
Comment (bhoflich) [inline weathervane.config.docker.micro.oneHost]: micro.oneHost probably does not need 8 drivers?

File: weathervane.config.docker.custom (hunk 1 above)
Comment (hrosenbe) [reply weathervane.config.docker.custom]: Yes.  This config file is just intended to show the variety of things you can specify.  I wouldn't expect anyone to use it this way.

File: weathervane.config.docker.micro.oneHost (hunk 2 above)
Comment (hrosenbe) [reply weathervane.config.docker.micro.oneHost]: Good catch.  I fixed this.

**Pull request merged**